File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/B. Even Numbers/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/B

using System;

namespace Codeforces.Assuit.Contest1
{
    class ProblemB
    {
        static void Main(string[] args)
        {
            ushort x = ushort.Parse(Console.ReadLine());
            ushort i = 0;
            if(x<2){
                Console.WriteLine(-1);
                return;
            }
            loop:
                Console.WriteLine(i+=2);
                if(i+1 < x) goto loop;
        }
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/B. Even Numbers/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/B
#include<iostream>
#include<cstdio>

using namespace std;
 
int main(){
    unsigned short x, i = 0;
    cin >> x;
    if(x<2){
        cout << -1 << endl;
        return 0;
    }
    loop:
        cout << (i+=2) << endl;
        if(i+1 < x) goto loop;
}